lib/migration.pm

```diff
@@ -118,6 +120,15 @@ sub remove_ltss {
         }
         set_var('SCC_ADDONS', join(',', grep { $_ ne 'ltss' } @$scc_addons));
     }
+    if (get_var('DROPPED_MODULES', '')) {
+        my ($droplist) = @_;
+        $droplist //= get_var('DROPPED_MODULES');
+        my @names = split(/,/, $droplist);
+        foreach my $name (@names) {
+            record_info "remove $name", 'some modules are dropped in target product, now remove them before migration';
+            remove_suseconnect_product(get_addon_fullname($name));
+        }
```

Comment:
After drop some modules, you also need reset SCC_ADDONS setting like ltss: set_var('SCC_ADDONS', join(',', grep { $_ ne 'ltss' } @$scc_addons));

Reply:
Add it now, thanks for review.